clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -119,15 +119,15 @@
 public interface ConsumerRebalanceListener {
 
     /**
-     * A callback method the user can implement to provide handling of offset commits to a customized store.
+     * A callback method the user can implement to provide handling of offset commits **sent to** a customized store.
```

Comment:
I'm not sure that "sent to" adds that much.

Maybe:

```suggestion
     * A callback method the user can implement to provide handling of offset commits **stored in** a customized store.
```

Or:

```suggestion
     * A callback method the user can implement to provide handling of offset commits **managed in** a customized store.
```